VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 768 HEARTBEAT: 8 Vector__XXX
 SG_ State : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Counter : 8|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 769 ERROR: 8 Vector__XXX
 SG_ APPS_Disagree : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ TrailBraking_Active : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ BPPS_Disagree : 16|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 770 DIGITAL: 8 Vector__XXX
 SG_ Precharge_Button : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ Drive_Button : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ DriveState_Request : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ RTDS : 24|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 772 ANALOGUE: 8 Vector__XXX
 SG_ APPS_1_Percent : 0|8@1+ (1,0) [0|0] "%" Vector__XXX
 SG_ APPS_2_Percent : 8|8@1+ (1,0) [0|0] "%" Vector__XXX
 SG_ APPS_Avg_Percent : 16|8@1+ (1,0) [0|0] "%" Vector__XXX

BO_ 351 BRAKE_MODULE_CALIBRATION: 8 Vector__XXX
 SG_ Brake_1_min : 7|16@0+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake_1_max : 23|16@0+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake_2_min : 39|16@0+ (1,0) [0|0] "" Vector__XXX
 SG_ Brake_2_max : 55|16@0+ (1,0) [0|0] "" Vector__XXX

BO_ 595 RMS_COMMAND_MESSAGE: 8 Vector__XXX




